blazor/ServerSideExample/ServerSideExample/ViewModels/CounterViewModel.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Reactive;
+using System.Threading.Tasks;
+using ReactiveUI;
+
+namespace ServerSideExample.ViewModels
+{
+    public class CounterViewModel : ReactiveObject
+    {
+        private int _currentCount;
+
+        public CounterViewModel()
+        {
+            Increment = ReactiveCommand.CreateFromTask(IncrementCount);
+        }
+
+        public int CurrentCount
+        {
+            get => _currentCount;
+            set => this.RaiseAndSetIfChanged(ref _currentCount, value);
+        }
+        
+        public ReactiveCommand<Unit, Unit> Increment { get; }
+
+        private Task IncrementCount()
+        {
+            _currentCount++;
```

Comment:
Mutating `_currentCount` property won't affect the UI here, as there is no call to `RaiseAndSetIfChanged`. Probably it should be `CurrentCount++`? Or, if `CurrentCount` mutation happens only through a command, worth declaring the property as `ObservableAsPropertyHelper<int>` and the command as `ReactiveCommand<Unit, int>` etc.

Reply:
It does affect the UI, though. On all three plaforms, Server, Wasm and Hosted.

Reply:
Updated in line with your comment.  CounterViewModel now uses ObservableAsPropertyHelper piped from ReactiveCommand<Unit, int>.

Retested.  Works on all three platforms.

Reply:
done